vendor/bat-native-ads/src/bat/ads/internal/ads/serving/choose/sample_ads.h

```diff
@@ -16,17 +17,28 @@
 
 namespace ads {
 
+template <typename T>
+T CalculateNormalizingConstantBase(const std::vector<T> scores) {
+  T normalizing_constant = 0.0;
```

Comment:
what if T was an int?

Reply:
would `T normalizing_constant = 0;` work for both `int` and `float`?